prober.go

```diff
@@ -60,6 +61,10 @@ func (p *prober) AddHTTP(id string, probingInterval time.Duration, endpoints []s
 					panic(err)
 				}
 				resp, err := p.tr.RoundTrip(req)
+				if err == nil && resp.StatusCode != http.StatusOK {
+					err = fmt.Errorf("got %s from %s", resp.Status, endpoints[pinned])
```

Comment:
got unexpected HTTP status code xxx from xxx.

Reply:
Good point, updated now. Thanks for the review.